kpops/components/streams_bootstrap/streams/model.py

```diff
@@ -183,6 +188,16 @@ class StreamsAppAutoScaling(CamelCaseConfigModel, DescConfigModel):
     )
     model_config = ConfigDict(extra="allow")
 
+    @model_validator(mode="after")
+    def validate_model_config(self) -> Self:
+        if self.enabled and (not self.consumer_group or not self.lag_threshold):
```

Comment:
this will fail if `lag_threshold=0` (perhaps include in your test cases?)

Reply:
True, I can change this to:
```suggestion
        if self.enabled and (self.consumer_group is None or not self.lag_threshold is None):
```

Reply:
yup, also remove the `not`, otherwise looks good